docs/sources/live/configure-grafana-live.md

```diff
@@ -0,0 +1,115 @@
++++
+title = "Configure Grafana Live"
+description = "Grafana Live configuration guide"
+keywords = ["Grafana", "live", "guide", "websocket"]
+weight = 120
++++
+
+# Configure Grafana Live
+
+Grafana Live is enabled by default. As of Grafana v8.0, it has a strict default for a maximum number of connections per Grafana server instance. 
+
+## Max number of connections
+
+Grafana Live uses persistent connections (WebSocket at the moment) to deliver real-time updates to clients. You may need to tune your server infrastructure a bit.
+
+The number of maximum connections users can establish with Grafana is limited to 100:
+
+```
+[live]
```

Comment:
Is this in the config file? If so, you should link to it.

Reply:
Fixed in e0718670d0ca16371a7a0afbf467d493448f31a0, actually I have not seen we have a detailed configuration document under administration - added Live section to it.